aws/table_aws_ssm_document.go

```diff
@@ -359,6 +373,62 @@ func getAwsSSMDocument(ctx context.Context, d *plugin.QueryData, h *plugin.Hydra
 	return data.Document, nil
 }
 
+func getAwsSSMDocumentContent(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+
+	var arn string
+	item, err := getAwsSSMDocumentArn(ctx, d, h)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_ssm_document.getAwsSSMDocumentContent", "arn_formatting_error", err)
```

Comment:
What is the possible `arn_formatting_error`? Could you please mention the error that is returned?

Reply:
The `getAwsSSMDocumentArn` function returns `(interface{}, error)`. If an error occurs within this function, we log it for debugging purposes. However, we do not have specific information about the type of error that can be returned. It could be a nil pointer dereference error, typecasting error, or any other type of error.